be/src/exec/pipeline/operator.h

```diff
@@ -82,19 +91,36 @@ class Operator {
 
     RuntimeProfile* get_runtime_profile() const { return _runtime_profile.get(); }
 
-    std::string get_name() const {
-        std::stringstream ss;
-        ss << _name + "_" << this;
-        return ss.str();
+    virtual std::string get_name() const {
+        return strings::Substitute("$0_$1($2)", _name, this, is_finished() ? "X" : "O");
     }
 
+    const LocalRFWaitingSet& rf_waiting_set() const;
+
+    RuntimeFilterHub* runtime_filter_hub();
+
+    std::vector<ExprContext*>& runtime_in_filters();
+
+    RuntimeFilterProbeCollector* runtime_bloom_filters();
+
+    // equal to ExecNode::eval_conjuncts(_conjunct_ctxs, chunk), is used to apply in-filters to Operators.
+    void eval_conjuncts_and_in_filters(const std::vector<ExprContext*>& conjuncts, vectorized::Chunk* chunk);
+    // equal to ExecNode::eval_join_runtime_filters, is used to apply bloom-filters to Operators.
+    void eval_runtime_bloom_filters(vectorized::Chunk* chunk);
+    void eval_runtime_bloom_filters(vectorized::ChunkPtr* chunk);
```

Comment:
Remove it

Reply:
this three functions are required.

Reply:
I think if we have eval_runtime_bloom_filters(vectorized::Chunk* chunk), we needn't have  void eval_runtime_bloom_filters(vectorized::ChunkPtr* chunk);

Reply:
ok